src/ephemera/bullets.js

```diff
@@ -0,0 +1,19 @@
+import throttle from 'lodash/fp/throttle'
+import {CONFIG} from '../config'
+import {WORLD} from '../world'
+import {Circle} from '../util'
+
+const {bullets} = CONFIG
+const {fireRate} = bullets // bullets per this as milli
+export const fireBullet = throttle(fireRate, () => {
+  const {player} = WORLD
+  const {physics} = player
+  const {body} = physics
+  const b = Circle.of({
+    x: body.pos.x + body.size.x / 2,
+    y: body.pos.y - 15,
+    a: 5
```

Comment:
Right now the `a` param is a holdover from old code; will likely augment the definition so it takes a `radius` / `_radius` param so the same code can be used in different parts of the pipeline.